### DISCUSSION TITLE
Min Max Avg

### DISCUSSION THREAD

**Discussion by WillianCamargo01** [alexjustesen/speedtest-tracker#1708] [Ideas]
Insert in addition to the last download and last upload also maximum, minimum and average

**Comment by SpiderMANek** [comment]
> Insert in addition to the last download and last upload also maximum, minimum and average

Totally agreed. These are very useful data.

**Comment by svenvg93** [comment]
Average will come in https://github.com/alexjustesen/speedtest-tracker/pull/1669 

Min and Max might come in a later stage, when we find a nice way of showing it
